app/models/spree/order_decorator.rb

```diff
@@ -1,5 +1,7 @@
 module SpreeMultiVendor
   module OrderDecorator
+    Spree::Order.state_machine.after_transition to: :complete, do: :send_notification_mails_to_vendors
```

Comment:
@aplegatt please do it the same way as here: https://github.com/spree-contrib/spree_multi_vendor/blob/master/app/models/spree/order_decorator.rb#L5